src/Bravo3/Orm/Drivers/Redis/RedisDriver.php

```diff
@@ -502,4 +511,42 @@ public function clearRefs($key)
     {
         $this->unit_of_work->addCommand('KeyDelete', [$key]);
     }
+
+    /**
+     * A wrapper function to wrap Redis commands which go to Predis Client
+     * in order to replay them if server connection issues occur.
+     *
+     * @param  string          $cmd function to call against the Predis client
+     * @param  array|callable  $params
+     * @param  int             $retry_iteration
+     * @return mixed
+     */
+    private function clientCmd($cmd, $params, $retry_iteration = 1) {
+        static $retry_delay = 200;
+
+        try {
+
+            if (1 > $retry_iteration) {
```

Comment:
```$retry_delay``` fixed.

Cant set $retry_iteration to 0 because in that case (0) the $retry_delay will be 0 too.